.github/workflows/ubuntu_build.yml

```
@@ -0,0 +1,26 @@
+# https://github.com/motioneye-project/motioneye/wiki/%28Install-On-Ubuntu-%2820.04-or-Newer%29
+name: ubuntu_build
+on: [pull_request, push]
+jobs:
+  ubuntu_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo apt-get -q update
+      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq upgrade
+      - run: sudo DEBIAN_FRONTEND=noninteractive apt-get -qq install
+             curl ffmpeg libcurl4-openssl-dev libffi-dev libjpeg-dev libssl-dev
+             libzbar-dev libzbar0 motion python3-dev python3-pip ssh v4l-utils
+      # - run: python3 -m pip install https://github.com/motioneye-project/motioneye/archive/dev.zip
+      - run: sudo python3 -m pip install .  # Must be sudo.  See #1371
+      - run: sudo mkdir -p /etc/motioneye
+      - run: sudo cp motioneye/extra/motioneye.conf.sample /etc/motioneye/motioneye.conf
+      - run: sudo mkdir -p /var/lib/motioneye
+      - run: sudo cp motioneye/extra/motioneye.systemd-unit-local /etc/systemd/system/motioneye.service
+      - run: sudo systemctl daemon-reload
+      - run: sudo systemctl enable motioneye
+      - run: sudo systemctl start motioneye
+      # Docker on GitHub Actions is not exporting port 8765 but that is OK for this test.
+      - run: sudo netstat -tulpn  # | grep 8765
```

```suggestion
      - run: ss -tln | grep 8765
```
But this may fail directly after the service start. We could add a loop with a timeout:
```sh
i=0
until ss -tln | grep 8765; do sleep 1; i=$(expr $i + 1); [ $i -le 10 ] || exit 1; done
```